front-end/src/domains/GpuServerInfoItem/GpuServerInfoItem.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import GpuServerInfoItem from "./GpuServerInfoItem";
+
+export default {
+  title: "Domains/GpuServerInfoItem",
```

Comment:
```suggestion
  title: "Domains/GpuServer/InfoItem",
```

이렇게 하면 어떨까요? 스토리북에서 볼 때 도메인끼리 폴더링이 되어서 보기 편하더라구요 ㅎㅎ

![image](https://user-images.githubusercontent.com/31029000/125480801-e385d166-58c2-4f49-a088-5a0959469736.png)

Reply:
적용